fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImplWithHeaderNestingSpec.groovy

```diff
@@ -50,7 +51,8 @@ class AsyncDruidWebServiceImplWithHeaderNestingSpec extends Specification {
                                 "2016-12-25T00:00:00.000Z/2017-01-03T00:00:00.000Z"
                             ],
                             "uncoveredIntervalsOverflowed": true
-                        }
+                        },
+                        "status-code": 200
```

Comment:
it looks to me like this is a test of the HeaderNestingJsonBuilderStrategy, not of the webservice itself.